### DISCUSSION TITLE
Make Login? How to identify and process depending on what the client says?

### DISCUSSION THREAD

**Discussion by SnowdenDev** [jchristn/WatsonTcp#272] [Q&A] [answered]
### **I have a server and a client, the client sends the username and password of a login to the server, how do I make the server recognize that it is a login and execute a function?

I understand that using dictionary receives a Sting and an object, so I don't understand how to use this on the server and the client, check the examples, but I can't find what I'm looking for!**

**Comment by jchristn** [comment, marked as answer]
Hi @SnowdenDev how are you sending the username and password?  Best practice would be similar to what I mentioned in the other discussion [here](https://github.com/dotnet/WatsonTcp/discussions/269).  I see you added a comment, so I'll go back and reply there!
